locale/areas.ru.tr

```diff
@@ -32,7 +32,7 @@ Give a player access to a sub-area between two positions that have already been
 Invalid regular expression.=Неверное регулярное выражение.
 Limit: @1 areas=Ограничение: @1 территорий(я)
 Limit: no area count limit=Ограничение: нет ограничений на кол-во территорий
-List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий (или всех существующих территорий, если Вы имеете привилегию areas)
+List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий или всех существующих территорий, если вы администратор.
```

Comment:
here is not truth, as I see in code.
maybe I'm wrong, but

you may not be an administrator, but have the privilege area

Reply:
@ssdaniel24 , can you double check?

Reply:
it could be just a mistake. Why not to fix this?